--- konsole-4.11.0/src/TerminalDisplay.cpp~	2013-07-10 00:49:37.000000000 +0200
+++ konsole-4.11.0/src/TerminalDisplay.cpp	2013-08-08 13:19:44.501329893 +0200
@@ -317,7 +317,7 @@
     , _middleClickPasteMode(Enum::PasteFromX11Selection)
     , _scrollbarLocation(Enum::ScrollBarRight)
     , _scrollFullPage(false)
-    , _wordCharacters(":@-./_~")
+    , _wordCharacters(":@-./_~?=&%+#")
     , _bellMode(Enum::NotifyBell)
     , _allowBlinkingText(true)
     , _allowBlinkingCursor(false)
